template/istqb.cls

```diff
@@ -53,6 +53,117 @@
 \usepackage{parskip}  % Put half of line height between paragraphs.
 \sloppy  % Prevent overfull lines.
 
+%% Language
+\PassOptionsToPackage{shorthands=off}{babel}
+\RequirePackage[czech,slovak,english]{babel}
+\ExplSyntaxOn
+\tl_new:N \g_istqb_language_tl
+\prop_new:N \g_istqb_reference_labels_prop
+\cs_new:Nn
+  \istqb_set_language:n
+  {
+    \tl_gset:Nn
+      \g_istqb_language_tl
+      { #1 }
+    \prop_gclear:N
+      \g_istqb_reference_labels_prop
+    \str_case:nnF
+      { #1 }
+      {
+        { cs } {
+          \selectlanguage { czech }
+          \gdef \istqbstandardsname { Standardy }
+          \gdef \istqbdocumentsname { ISTQB\textsuperscript{\textregistered}~dokumenty }
+          \gdef \istqbbooksname { Knihy }
+          \gdef \istqbarticlesname { Články }
+          \gdef \istqbfurtherreadingname { Další~zdroje }
+          \gdef \istqbpagename { Strana }
+          \gdef \istqboutofname { z }
+          \gdef \istqbprovidedbysingularname { Poskytl }
+          \gdef \istqbprovidedbypluralname { Poskytli }
+          \gdef \istqborgname { International~Software~Testing~Qualifications~Board }
+          \gdef \istqblistdelimiters
+            {
+              { ~a~ }
+              { ,~ }
+              { ~a~ }
+            }
+          \prop_gset_from_keyval:Nn
+            \g_istqb_reference_labels_prop
+            {
+              section = { Sekci },
+              figure = { \figurename },
+              table = { Tabulku },
+            }
+        }
+        { sk } {
+          \selectlanguage { slovak }
+          \gdef \istqbstandardsname { Štandardy }
+          \gdef \istqbdocumentsname { ISTQB\textsuperscript{\textregistered}~dokumenty }
+          \gdef \istqbbooksname { Knihy }
+          \gdef \istqbarticlesname { Články }
+          \gdef \istqbfurtherreadingname { Ďalšie~zdroje }
+          \gdef \istqbpagename { Strana }
+          \gdef \istqboutofname { z }
+          \gdef \istqbprovidedbysingularname { Poskytol }
+          \gdef \istqbprovidedbypluralname { Poskytli }
+          \gdef \istqborgname { International~Software~Testing~Qualifications~Board }
+          \gdef \istqblistdelimiters
+            {
+              { ~a~ }
+              { ,~ }
+              { ~a~ }
+            }
+          \prop_gset_from_keyval:Nn
+            \g_istqb_reference_labels_prop
+            {
+              section = { Sekcii },
```

Comment:
```suggestion
              section = { Sekciu },
```

Reply:
I fixed the Slovak translation of "Section" in commit ba225c905c82e0b27022fa41b63af94cafd04a33 from this PR.